sh-runtime.c

```diff
@@ -112,105 +108,6 @@ END_RUNTIME_FUN(local_vars)
-  putstr("    113) __char=\"q\" ;;\n");
-  putstr("    114) __char=\"r\" ;;\n");
-  putstr("    115) __char=\"s\" ;;\n");
-  putstr("    116) __char=\"t\" ;;\n");
-  putstr("    117) __char=\"u\" ;;\n");
-  putstr("    118) __char=\"v\" ;;\n");
-  putstr("    119) __char=\"w\" ;;\n");
-  putstr("    120) __char=\"x\" ;;\n");
-  putstr("    121) __char=\"y\" ;;\n");
-  putstr("    122) __char=\"z\" ;;\n");
-  putstr("    65)  __char=\"A\" ;;\n");
-  putstr("    66)  __char=\"B\" ;;\n");
-  putstr("    67)  __char=\"C\" ;;\n");
-  putstr("    68)  __char=\"D\" ;;\n");
-  putstr("    69)  __char=\"E\" ;;\n");
-  putstr("    70)  __char=\"F\" ;;\n");
-  putstr("    71)  __char=\"G\" ;;\n");
-  putstr("    72)  __char=\"H\" ;;\n");
-  putstr("    73)  __char=\"I\" ;;\n");
-  putstr("    74)  __char=\"J\" ;;\n");
-  putstr("    75)  __char=\"K\" ;;\n");
-  putstr("    76)  __char=\"L\" ;;\n");
-  putstr("    77)  __char=\"M\" ;;\n");
-  putstr("    78)  __char=\"N\" ;;\n");
-  putstr("    79)  __char=\"O\" ;;\n");
-  putstr("    80)  __char=\"P\" ;;\n");
-  putstr("    81)  __char=\"Q\" ;;\n");
-  putstr("    82)  __char=\"R\" ;;\n");
-  putstr("    83)  __char=\"S\" ;;\n");
-  putstr("    84)  __char=\"T\" ;;\n");
-  putstr("    85)  __char=\"U\" ;;\n");
-  putstr("    86)  __char=\"V\" ;;\n");
-  putstr("    87)  __char=\"W\" ;;\n");
-  putstr("    88)  __char=\"X\" ;;\n");
-  putstr("    89)  __char=\"Y\" ;;\n");
-  putstr("    90)  __char=\"Z\" ;;\n");
-  putstr("    32)  __char=\" \" ;;\n");
-  putstr("    33)  __char=\"!\" ;;\n");
-  putstr("    34)  __char=\"\\\"\" ;;\n");
-  putstr("    35)  __char=\"#\" ;;\n");
-  putstr("    36)  __char=\"$\" ;;\n");
-  putstr("    37)  __char=\"%\" ;;\n");
-  putstr("    38)  __char=\"&\" ;;\n");
-  putstr("    39)  __char=\"'\" ;;\n");
-  putstr("    40)  __char=\"(\" ;;\n");
-  putstr("    41)  __char=\")\" ;;\n");
-  putstr("    42)  __char=\"*\" ;;\n");
-  putstr("    43)  __char=\"+\" ;;\n");
-  putstr("    44)  __char=\",\" ;;\n");
-  putstr("    45)  __char=\"-\" ;;\n");
-  putstr("    46)  __char=\".\" ;;\n");
-  putstr("    47)  __char=\"/\" ;;\n");
-  putstr("    58)  __char=\":\" ;;\n");
-  putstr("    59)  __char=\";\" ;;\n");
-  putstr("    60)  __char=\"<\" ;;\n");
-  putstr("    61)  __char=\"=\" ;;\n");
-  putstr("    62)  __char=\">\" ;;\n");
-  putstr("    63)  __char=\"?\" ;;\n");
-  putstr("    64)  __char=\"@\" ;;\n");
-  putstr("    91)  __char=\"[\" ;;\n");
-  putstr("    92)  __char=\"\\\\\" ;;\n");
-  putstr("    93)  __char=\"]\" ;;\n");
-  putstr("    94)  __char=\"^\" ;;\n");
-  putstr("    95)  __char=\"_\" ;;\n");
-  putstr("    96)  __char=\"\\`\" ;;\n");
-  putstr("    123) __char=\"{\" ;;\n");
-  putstr("    124) __char=\"|\" ;;\n");
-  putstr("    125) __char=\"}\" ;;\n");
-  putstr("    126) __char=\"~\" ;;\n");
-  putstr("    10)  __char=\"\\n\" ;;\n");
-  putstr("    *)\n");
-  putstr("      echo \"Invalid character code: $1\" ; exit 1\n");
```

Comment:
As a proof that this function wasn't very useful (nor well tested), it doesn't even work for all characters!